akka-projection-grpc-tests/src/test/scala/akka/projection/grpc/internal/ProtoAnySerializationSpec.scala

```diff
@@ -21,7 +21,15 @@ import org.scalatest.wordspec.AnyWordSpecLike
 import com.google.protobuf.any.{ Any => ScalaPbAny }
 import com.google.protobuf.{ Any => PbAny }
 
-class ProtoAnySerializationSpec extends ScalaTestWithActorTestKit with AnyWordSpecLike with LogCapturing {
+class ProtoAnySerializationSpec
+    extends ScalaTestWithActorTestKit("""
+    akka.actor.serialization-bindings {
+      # // FIXME important to document this for the SerializedEvent optimization to work
+      "scalapb.GeneratedMessage" = proto
```

Comment:
An idea is that we add something like `optional-serialization-bindings` so that we can bind this by default in Akka's reference.conf without failing if scalapb is not in the classpath.